lib/presentation/widget/upload_request_inside/upload_request_inside_viewmodel.dart

```diff
@@ -73,25 +74,42 @@ class UploadRequestInsideViewModel extends BaseViewModel {
             (success) {
               if(success is UploadRequestViewState) {
                 if(_arguments.uploadRequestGroup.collective) {
-                  // List of files in Collective Upload request
-                  return store.dispatch(_getAllUploadRequestEntries(success.uploadRequests));
+                  _loadListFilesCollectiveUploadRequest(success);
                 } else {
-                  // List of recipients in Individual Upload request
-                  store.dispatch(GetAllUploadRequestsAction(Right(success)));
+                  if(_arguments.documentType == UploadRequestDocumentType.root) {
```

Comment:
IMO, root and children is not make sense for `UploadRequest`. How about:
```
enum UploadRequestDocumentType {recipients, files}
```

Reply:
agree, it looks better